Funcky.Test/Monads/OptionExtensionsTest.cs

```diff
@@ -20,6 +27,28 @@ public void WithToEitherYouCanConvertAnOptionToAnEitherWithAnIndividualLeftIfOpt
         fullUser.Switch(left: _ => Assert.Fail("failed"), right: firstName => Assert.Equal("Name", firstName));
     }
 
+    [Property]
+    public Property AnOptionWithAValueTypeCanBeConvertedToANullable(Option<int> input)
+    {
+        var nullable = input.ToNullable();
+
+        return input.Match(
+            none: () => nullable is null,
+            some: value => nullable == value)
+            .ToProperty();
```

Comment:
You're not actually testing that None maps to null and Some(x) to x. Is that intentional?

Reply:
I think I do:

let's say `input` is `Some(42) `
* leads to call of some with `value ` with the value `42`.
* I compare this with nullable which is also 42.

if the input is None
* leads to a call of none: in the Match, and I check if that `nullable is null`.